e2e-tests/cypress/integration/common/guidance-pages.js

```diff
@@ -0,0 +1,13 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+When('the appellant navigates to the next page', () => {
+  cy.guidancePageNavigation('next');
+});
```

Comment:
Do we need to confirm what page has been navigated to?

Reply:
This is just doing the page navigation, plus the page navigation confirmation is in the Then step